docker/web-and-data/Dockerfile

```diff
@@ -45,3 +48,6 @@ RUN find /cbioportal/core/src/main/scripts/ -type f -executable \! -name '*.pl'
 
 # put config files in this folder if you want to override config
 ENV PORTAL_HOME=/cbioportal
+
+RUN mkdir -p $PORTAL_WEB_HOME
+RUN unzip /app.war -d  $PORTAL_WEB_HOME
```

Comment:
Would it be an improvement to be consistent with the alternative Dockerfile below, and use the `jar` command?

Reply:
This command is not present in that image. I checked.

Reply:
@pvannierop @fedde-s Looks indeed like jre has no jar binary. It's only included in jdk images. It does have unzip included in the image so that's a good alternative